stable/prometheus-operator/README.md

```diff
@@ -307,6 +307,8 @@ The following tables list the configurable parameters of the prometheus-operator
 | `grafana.sidecar.datasources.defaultDatasourceEnabled` | Enable Grafana `Prometheus` default datasource` | `true` |
 | `grafana.sidecar.datasources.label` | If the sidecar is enabled, configmaps with this label will be loaded into Grafana as datasources configurations | `grafana_datasource` |
 | `grafana.rbac.pspUseAppArmor` | Enforce AppArmor in created PodSecurityPolicy (requires rbac.pspEnabled) | `true` |
+| `grafana.rbac.create` | Create rbac for grafana | `true` |
```

Comment:
Generally chats don't document sub-chart variables. If you document one, should you document them all?

Reply:
@vsliouniaev are you suggesting me to document all of them?

Reply:
I'm suggesting that you omit documenting them

Reply:
@vsliouniaev ack